packages/repository/test/unit/repository-mixin/repository-mixin.test.ts

```diff
@@ -42,6 +42,39 @@ describe('RepositoryMixin', () => {
     expectNoteRepoToBeBound(myApp);
   });
 
+  it('mixed class has .dataSource()', () => {
```

Comment:
The source code line defining a test should read like an English sentence. Right now, it does not: "it mixed class has .dataSource()".

I am proposing the following changes:

 - add a new `describe` block to group all tests related to `app.dataSource()` method.
 - rename this test to `it('is mixed into the target class')`

```ts
describe('dataSource()', () => {
  it('is mixed into the target class', () => {
    // ...
  });

  it('binds dataSource using dataSource name for the binding key', () => {
    // ...
  });

  it('binds dataSource using the provided binding key', () => {
    // ...
  });
});
```

Reply:
I was too lazy.
Let me refactor it.